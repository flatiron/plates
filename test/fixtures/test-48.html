<a data-bind-href="zero" data-bind="test">link</a>
