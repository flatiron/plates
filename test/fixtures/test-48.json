{
  "zero": "/sub-zero",
  "one": "Super link"
}
