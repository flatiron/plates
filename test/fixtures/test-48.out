<a data-bind-href="zero" data-bind="test" href="/sub-zero">Super link</a>
